chore(99designs/aws-vault): support darwin

u-yas opened the pull request:
Now that we have support for the dmg format, we have also added it to the aqua-registry

https://github.com/aquaproj/aqua/blob/main/tests/macos/registry.yaml#L8-L23

u-yas committed:
support darwin

suzuki-shunsuke commented:
Thank you for your contribution!

suzuki-shunsuke commented:
📝 aqua >= v2.2.0 is required. https://github.com/aquaproj/aqua/releases/tag/v2.2.0 https://github.com/aquaproj/aqua/issues/1706

suzuki-shunsuke committed:
fix(99designs/aws-vault): run aqua-registry gr

```console
$ aqua-registry gr
```

suzuki-shunsuke commented:
Please don't forget to update `registry.yaml` by `aqua-registry gr` command.

```console
$ aqua-registry gr
```

suzuki-shunsuke committed:
ci: use macos-latest for darwin/arm64

hdiutil is required

https://github.com/aquaproj/aqua-registry/pull/11204#issuecomment-1491120550

suzuki-shunsuke commented:
In this repository's CI, we simulate darwin/arm64 on linux/amd64, but hdiutil isn't installed in linux so CI failed.
https://github.com/aquaproj/aqua-registry/actions/runs/4570150891/jobs/8067155385?pr=11204

suzuki-shunsuke committed:
fix(99designs/aws-vault): re-generate by --deep option

```
rm pkgs/99designs/aws-vault/pkg.yaml pkgs/99designs/aws-vault/registry.yaml
aqua-registry scaffold --deep 99designs/aws-vault
```

suzuki-shunsuke commented:
Recreated configuration by `--deep` option. 

```sh
rm pkgs/99designs/aws-vault/pkg.yaml pkgs/99designs/aws-vault/registry.yaml
aqua-registry scaffold --deep 99designs/aws-vault
```

suzuki-shunsuke commented:
Released. https://github.com/aquaproj/aqua-registry/releases/tag/v3.149.1

u-yas commented:
@suzuki-shunsuke 

sorry. be careful from next time
